### PR TITLE
Storybook fixes & cleanup

### PR DESCRIPTION
**Pull request by OllysCoding**
## What does this change?

Update to use source resets, across storybook, AR, and DCR. As AR was not using any resets before, some changes had to be made to support them.

The following fixes were applied

- Fix issues introduced to AR by using a CSS reset
- Remove white background on storybook which breaks some DCR stories
- Fix fonts for AR in storybook (?)

## Why?

Prior to this change, resets were applied to AR in storybook, but not in production, meaning that stories often displayed incorrectly. By moving the resets to source we've introduced a more robust standard for resets across the guardian, where previously they were only used in DCR.

### CODE REVIEW

**Review by lucymonie — COMMENTED** [review]
This looks good to me but I will leave for AR people to approve. Love all the deletions, though it's always weird to see someone adding `box-sizing: content-box`!

File: dotcom-rendering/src/amp/server/document.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import createEmotionServer from '@emotion/server/create-instance';
 import createCache from '@emotion/cache';
 import he from 'he';
 
-import resetCSS from /* preval */ '@root/src/lib/reset-css';
+import { resets } from '@guardian/src-foundations/utils';
```
Comment (oliverlloyd) [inline dotcom-rendering/src/amp/server/document.tsx:8]: What was `preval` doing before?

File: dotcom-rendering/src/web/server/htmlTemplate.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import resetCSS from /* preval */ '@root/src/lib/reset-css';
+import { resets } from '@guardian/src-foundations/utils';
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/server/htmlTemplate.ts:1]: Ditto on preval

**Review by oliverlloyd — COMMENTED** [review]
Looks great! Thanks for the extra Storybook ❤️ 

Just a question on preval. I am not aware what that was doing but I'm guessing it was a performance optimisation?

File: dotcom-rendering/src/amp/server/document.tsx (hunk 1 above)
Comment (OllysCoding) [reply dotcom-rendering/src/amp/server/document.tsx:8]: Preval was used to read the file with `fs.readFileSync` and then spit it out into a string to be used a runtime

Comment (OllysCoding) [reply dotcom-rendering/src/amp/server/document.tsx:8]: Now it's in source it's already in a string

File: apps-rendering/src/components/byline.tsx [hunk 3]
```diff
@@ -22,6 +22,7 @@ interface Props extends Format {
 const styles = (kicker: string): SerializedStyles => css`
 	${headline.xxxsmall()}
 	color: ${kicker};
+	font-style: italic;
```
Comment (JamieB-gu) [inline apps-rendering/src/components/byline.tsx:25]: I believe Source exposes this as part of the typography API? So you can do:

```ts
headline.xxxsmall({ fontStyle: 'italic' })
```

File: apps-rendering/src/components/byline.tsx [hunk 4]
```diff
@@ -81,6 +82,7 @@ const commentAnchorStyles = (
 const labsStyles = css`
 	${textSans.medium({ lineHeight: 'regular' })}
 	color: ${palette.labs[300]};
+	font-style: italic;
```
Comment (JamieB-gu) [inline apps-rendering/src/components/byline.tsx:85]: Same comment about the typography API.

File: package.json [hunk 5]
```diff
@@ -24,11 +24,21 @@
...
+    "@guardian/src-button": "^3.10.0",
+    "@guardian/src-checkbox": "^3.10.0",
+    "@guardian/src-foundations": "^3.10.0",
+    "@guardian/src-icons": "^3.10.0",
+    "@guardian/src-link": "^3.10.0",
+    "@guardian/src-radio": "^3.10.0",
+    "@guardian/src-text-area": "^3.10.0",
+    "@guardian/src-text-input": "^3.10.0",
+    "@guardian/source-react-components-development-kitchen": "^0.0.3",
+    "@guardian/src-svgs": "^2.8.2",
```
Comment (JamieB-gu) [inline package.json:41]: 👍

**Review by JamieB-gu — APPROVED** [review]
This is great 🎉

**Review by oliverlloyd — APPROVED** [review]
Echoing @JamieB-gu comments on the italic text but otherwise 👍

**Pull request merged**